Lombiq.DataTables/Tests/Lombiq.DataTables.Tests/UnitTests/Services/ExportTests.cs

```diff
@@ -95,6 +96,13 @@ public class ExportTests
 
     public static IEnumerable<object[]> Data()
     {
+        // ClosedXML looks at the CurrentCulture to initialize the workbook's culture. They also to set it like this
+        // in their own unit tests. See:
+#pragma warning disable S103 // Split this long line.
+        // https://github.com/ClosedXML/ClosedXML/blob/c2d408b127844ea3d4a5f6b060c548c953b6bcf3/ClosedXML_Tests/Excel/CalcEngine/LookupTests.cs#L16-L17
+#pragma warning restore S103 // Split this long line.
+        Thread.CurrentThread.CurrentCulture = new CultureInfo("en-US", useUserOverride: false);
```

Comment:
You shouldn't worry, `Thread.CurrentThread.CurrentCulture` is thread safe so it can't conflict with others. See mini experiment:
![image](https://user-images.githubusercontent.com/4007293/168436898-ecf7d2d4-c036-4cca-b4cf-6964d9d92075.png)

I have moved the assignment inside the `ExportTableShouldMatchExpectation` just to be sure.

Reply:
Great!